pkg/lightning/backend/remote/cache_test.go

```diff
@@ -0,0 +1,67 @@
+// Copyright 2025 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package remote
+
+import (
+	"os"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestChunksCache(t *testing.T) {
+	testChunksCache(t, "")
+	basePath := os.TempDir()
+	testChunksCache(t, basePath)
+}
+
+func testChunksCache(t *testing.T, basePath string) {
```

Comment:
maybe add some negative cases, such as chunk not exist, you can run with coverage to found out